static/openapi/dem/elevation-openapi.yaml

```diff
@@ -106,18 +100,31 @@ components:
                 nullable: true
                 type: number
                 format: double
+              productId:
```

Comment:
Can you please change the example to show the correlation between the data productId to one of the returned products?

Reply:
Done, examples are provided